test/index.js

```diff
@@ -353,6 +381,24 @@ describe('KeyringController', function () {
       // was also removed after removing the account
       expect(keyringController.keyrings).toHaveLength(1);
     });
+
+    it('does not remove the keyring if there are accounts remaining after removing one from the keyring', async function () {
+      // Add a new keyring with one account
+      await keyringController.addNewKeyring('HD Key Tree', {
+        mnemonic: walletTwoSeedWords,
+        numberOfAccounts: 2,
```

Comment:
The comment says that we're adding a keyring with 1 account, but this is asking the keyring controller to generate 2 accounts